cost-events-audit-api.md

```diff
@@ -0,0 +1,115 @@
+# Cost Events Audit API
```

Comment:
@teevans @bstuder99 do we plan to create another docs page for the audit UI?

Reply:
Yes, WIP

Reply:
https://github.com/kubecost/docs/pull/574